devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -276,3 +298,29 @@ private command(physicalgraph.zwave.Command cmd) {
 private isEnerwave() {
 	zwaveInfo?.mfr?.equals("011A") && zwaveInfo?.prod?.equals("0601") && zwaveInfo?.model?.equals("0901")
 }
+
+private incrWakeUpRequestRefCount() {
+	if (state.wakeUpRequestRefCount == null) {
+		state.wakeUpRequestRefCount = 0
+	}
+
+	state.wakeUpRequestRefCount = state.wakeUpRequestRefCount + 1
+}
+
+private decrWakeUpRequestRefCount() {
+	def result = ""
```

Comment:
do empty event strings cause any trouble?

Reply:
So far in my testing: nope.

Reply:
I went ahead and wrapped it in a response() call just so I could guarantee that the function was following its promise.